pkgs/by-name/jj/jjui/package.nix

```
@@ -0,0 +1,34 @@
+{
+  pkgs,
+  lib,
+  ...
+}:
+pkgs.buildGoModule rec {
+  pname = "jjui";
+  version = "b1f467fcef89a6f9e57a5be2108bb0ef34f1442d";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "idursun";
+    repo = pname;
+    rev = version;
+    hash = "sha256-Ig4Amc9zc6rfwh4CyBhhk05/5Y8JpBT8OflxHXtTuOs=";
+  };
+
+  vendorHash = "sha256-kg5b3tzwyAhn00GwdUDf4OdYZvCJZHgkgpzHFWy5SxI=";
+
+  doCheck = false; # NOTE: Tests are disabled due to a bug in the tests upstream. Uncomment when the bug is fixed.
+
+  fixupPhase = ''
```

```suggestion
  postFixup = ''
```
We don't want to override the main phase here

---

This seems to work well and should be indeed a better place to modify rename the binary. Modified.